src/main/java/by/exadel/internship/controller/FormController.java

```diff
@@ -21,20 +22,20 @@
 @Api(tags = "Endpoints for Form")
 public class FormController {
 
-    private final FormService formService;
+    private final FormService FORM_SERVICE;
 
     @PostMapping(consumes = {MULTIPART_FORM_DATA_VALUE, APPLICATION_JSON_VALUE})
     @ApiOperation("Add new form")
     @ResponseStatus(HttpStatus.CREATED)
     public FormFullDTO addNewForm(@RequestPart(name = "form") FormRegisterDTO form,
                                   @RequestPart(name = "file", required = false) MultipartFile file) {
-        return formService.process(form, file);
+        return FORM_SERVICE.process(form, file);
     }
 
     @GetMapping
     @ApiOperation("Get all forms")
     @ResponseStatus(HttpStatus.OK)
-    public List<FormFullDTO> getAllForms() {
-        return formService.getAll();
+    public List<Form> getAllForms() {
```

Comment:
Why is it changes to `Form`?

Reply:
because it was to Valentin's branch ? but i changed it in my previouce pr, and now changed for it too. Thanks